tests/test_metrics.py

```diff
@@ -86,4 +89,8 @@ def test_metrics(farm, runner):
         assert count_forward[ind].value == N_CALLS
         assert farm.counters()[p][ENDPT_NAME_FWD] == N_CALLS
 
+    # Expect only queue wait time for "status", because that's what `/metrics` gets routed to
```

Comment:
No, it's because in this test the endpoint `/status` is called to test the metrics. `/metrics` is run by the prometheus "client", it's not a coco endpoint.

Reply:
Right, will correct.